README.md

```diff
@@ -69,6 +69,27 @@ Now close your project and run `pod install` from the command line. Then open th
 Build your project once (with `Cmd+B`) to update the frameworks known to Xcode. Now you can `import AnyMenu` in each class you want to use its features.
 Refer to [CocoaPods.org](https://cocoapods.org) for detailed / updates instructions.
 
+### Accio
+
+Add the following to your Package.swift:
+
+```swift
+.package(url: "git@github.com:Flinesoft/AnyMenu.git", .branch("stable")),
```

Comment:
git@github.com will only work for people who have an SSH key added, please use https. Also please use `.upToNextVersion(from: "")` instead of `.branch` even it doesn't work right now.